### PR TITLE
Use new module "project_analytic_line_view"

### PR DESCRIPTION
**Pull request by mdietrichc2c**
This PR follows on what was done in https://github.com/OCA/project-service/pull/107 : to avoid losing functionality in `project_billing_utils` , a dependency on the module `project_analytic_line_view` is now replacing the moved code.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ virtualenv:
 
 install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone https://github.com/OCA/project-service.git ${HOME}/project-service
```
Comment (yvaucher) [inline .travis.yml]: Could you add this in an oca_dependencies.txt file ?

Comment (yvaucher) [inline .travis.yml]: As an example:

https://github.com/OCA/carrier-delivery/blob/8.0/oca_dependencies.txt

**Comment by pedrobaeza** [comment]
:+1: with @yvaucher's remark

**Comment by mdietrichc2c** [comment]
@yvaucher @pedrobaeza done and rebased. Thanks!

**Comment by pedrobaeza** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:

**Pull request merged**